### ISSUE TITLE
LaTeX \sphinxupquote{} breaks in Brazilian

### ISSUE THREAD

**Issue by jdemeyer** [sphinx-doc/sphinx#4978]
Subject: after upgrading to Sphinx 1.7.4 from Sphinx 1.6.3, our Brazilian PDF documentation no longer compiles.

### Problem
The following ReST excerpt
```
-  EllipticCurve(label): Fornece a curva elíptica da base de dados
   Cremona com o "label" (novo) dado. O label é uma string, tal como
   ``"11a"`` ou ``"37b2"``. As letras devem ser minúsculas (para
   distinguir dos labels antigos).
```
gets compiled by Sphinx into the LaTeX
```
\item {}
EllipticCurve(label): Fornece a curva elíptica da base de dados
Cremona com o “label” (novo) dado. O label é uma string, tal como
\sphinxcode{\sphinxupquote{"11a"}} ou \sphinxcode{\sphinxupquote{"37b2"}}. As letras devem ser minúsculas (para
distinguir dos labels antigos).
```
The preamble contains
```
\documentclass[letterpaper,10pt,brazil]{sphinxmanual}
```
When `pdflatex` is run, the following error is produced:
```
! Argument of \language@active@arg" has an extra }.
<inserted text>
                \par
l.3444 \sphinxcode{\sphinxupquote{"11a"}} 
                                          ou \sphinxcode{\sphinxupquote{"37b...

?
! Emergency stop.
<inserted text> 
                \par
l.3444 \sphinxcode{\sphinxupquote{"11a"}} 
                                          ou \sphinxcode{\sphinxupquote{"37b...

!  ==> Fatal error occurred, no output PDF file produced!
Transcript written on tutorial-pt.log.
```

#### Expected results
Sphinx should generate LaTeX which actually compiles.

### Reproducible project / your project
- https://trac.sagemath.org/ticket/24935

### Environment info
- OS: `Linux sage4 4.4.26-gentoo #2 SMP Tue Nov 8 12:29:18 CET 2016 x86_64 Intel(R) Xeon(R) CPU X5660 @ 2.80GHz GenuineIntel GNU/Linux`
- Python version: `2.7.14`
- Sphinx version: `1.7.4`
- TeX version:
```
pdfTeX 3.14159265-2.6-1.40.18 (TeX Live 2017)
kpathsea version 6.2.3
Copyright 2017 Han The Thanh (pdfTeX) et al.
There is NO warranty.  Redistribution of this software is
covered by the terms of both the pdfTeX copyright and
the Lesser GNU General Public License.
For more information about these matters, see the file
named COPYING and the pdfTeX source.
Primary author of pdfTeX: Han The Thanh (pdfTeX) et al.
Compiled with libpng 1.5.7; using libpng 1.5.7
Compiled with zlib 1.2.8; using zlib 1.2.8
Compiled with poppler version 0.62.0
```

**Comment by jdemeyer** [comment]
The following post looks relevant: https://tex.stackexchange.com/questions/340661/argument-of-languageactivearg-has-an-extra-i-use-includegraphics-and-r

**Comment by tk0miya** [comment]
Fixed by #4984.
Thank you for reporting
